src/Shared/Microsoft.AspNetCore.Razor.Utilities.Shared/HashAlgorithmOperations.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Security.Cryptography;
+
+namespace System;
+
+internal static class HashAlgorithmOperations
+{
+    public static HashAlgorithm Create()
+        => SHA256.Create();
+    public static string? GetAlgorithmName()
+        => HashAlgorithmName.SHA256.Name;
+    public static string? GetOutdatedAlgorithmName()
+        => HashAlgorithmName.SHA1.Name;
```

Comment:
This method shouldn't exist IMO. If this class is going to help with allowing us to change the default algorithm, the idea of an "outdated" one doesn't make sense. Persumably in time there will be multiple outdated algorithms.